bob/src/cluster/operations.rs

```diff
@@ -30,6 +30,18 @@ fn call_node_put(
     tokio::spawn(task)
 }
 
+fn call_node_delete(
+    key: BobKey,
+    options: DeleteOptions,
+    node: Node,
+) -> JoinHandle<Result<NodeOutput<()>, NodeOutput<Error>>> {
+    debug!("DELETE[{}] delete to {}", key, node.name());
+    let task = async move {
+        LinkManager::call_node(&node, |conn| conn.delete(key, options).boxed()).await
```

Comment:
Tests build failed: `no method named 'delete' found for reference '&MockBobClient' in the current scope`. Looks like `delete` should be added to `MockBobClient`